lib/std/os/bits/windows.zig

```diff
@@ -256,6 +256,41 @@ pub const POLLERR = ws2_32.POLLERR;
 pub const POLLHUP = ws2_32.POLLHUP;
 pub const POLLNVAL = ws2_32.POLLNVAL;
 
+pub const SOL_SOCKET = 0xffff;
+
+pub const SO_DEBUG = 0x0001;
```

Comment:
Should these come from `ws2_32`?

Reply:
Yup, I keep losing track of the many places where bits go.